bridge_adaptivity/module/views.py

```diff
@@ -675,6 +675,24 @@ class SequenceComplete(LtiSessionMixin, DetailView):
     template_name = 'module/sequence_complete.html'
 
 
+def sync_collection(request, slug, api_request=None):
```

Comment:
Please clarify why was this change done?

Reply:
I wrong when I moved the  function `_check_and_build_web_socket_message`. I fixed it.